### DISCUSSION TITLE
No acces to benifits despite approval

### DISCUSSION THREAD

**Discussion by Killian-Conyngham** [community/community#119945] [GitHub Education]
### Hi everyone,

I sent off my application months ago, and on the status page it says both approved and rejected, and I do not have access to the benefits of the student development pack.

![image](https://github.com/community/community/assets/19914082/959d4652-175e-441f-b5e8-bc45cb2581cf)

**Comment by samus-aran** [comment]
Hi @Killian-Conyngham 

It appears you do have the academic benefits applied (hence your Pro account). What benefits are you not able to access for you believe they haven't been applied?

**Comment by Killian-Conyngham** [comment]
Apologies, I was confused as I could not figure out how to activate
copilot, but in the end when I applied for the trial my student status was
recognised.
